udata/harvest/backends/dcat.py

```diff
@@ -105,13 +112,15 @@ def get_format(self):
                 raise ValueError(msg)
         return fmt
 
-    def parse_graph(self, url, fmt) -> List[Graph]:
+    def walk_graph(self, url: str, fmt: str, do) -> List[Graph]:
```

Comment:
I think we could refactor to return a single graph now instead of one per page?
May be studied later on however.